### ISSUE TITLE
node_modules is not committed

### ISSUE THREAD

**Issue by dominikfoldi** [8BitJonny/gh-get-current-pr#2]
When I run the workflow I get the following error because the `node_modules` folder is not committed to the action's repository:

```
internal/modules/cjs/loader.js:800
    throw err;
    ^

Error: Cannot find module '@actions/core'
Require stack:
- /home/runner/work/_actions/8BitJonny/gh-get-current-pr/1.0.0/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:797:15)
    at Function.Module._load (internal/modules/cjs/loader.js:690:27)
    at Module.require (internal/modules/cjs/loader.js:852:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/home/runner/work/_actions/8BitJonny/gh-get-current-pr/1.0.0/index.js:7:44)
    at Module._compile (internal/modules/cjs/loader.js:959:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:995:10)
    at Module.load (internal/modules/cjs/loader.js:815:32)
    at Function.Module._load (internal/modules/cjs/loader.js:727:14)
    at Function.Module.runMain (internal/modules/cjs/loader.js:1047:10) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/runner/work/_actions/8BitJonny/gh-get-current-pr/1.0.0/index.js'
  ]
}
```

Could you please commit the node_modules as well?

**Comment by kbakk** [comment]
Weird as I find it, [documentation states that you should commit node_modules](https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github).

**Comment by 8BitJonny** [comment]
You are totally right.
I'm so sorry for my delayed response but for some reason I didn't get a notification for this

@kbakk your PR looks already quite good. Let me quickly add a small test case so that for the future such a bug doesn't happen unrecognised again.
But then we are good to go and ready to resolve this issue

**Comment by 8BitJonny** [comment]
Just released v1.0.1 with the bug fix

Thanks guys for the contribution @dominikfoldi @kbakk

**Comment by dominikfoldi** [comment]
Thank you @8BitJonny and @kbakk!
